From 3e78b7523f3293aa43c152d033cffca34ffb8119 Mon Sep 17 00:00:00 2001
From: Gunther Hagleitner <gunther@apache.org>
Date: Thu, 24 Sep 2015 15:20:26 -0700
Subject: [PATCH 0372/1164] HIVE-11932: JDBC Driver appends an extra / when
 configuring connection by reading httpPath from
 ZooKeeper (Vaibhav Gumashta, reviewed by Thejas
 Nair)

Change-Id: I189a77ace1fd34a81683df3ac748a24629c719f4
---
 .../hive/jdbc/ZooKeeperHiveClientHelper.java       |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/jdbc/src/java/org/apache/hive/jdbc/ZooKeeperHiveClientHelper.java b/jdbc/src/java/org/apache/hive/jdbc/ZooKeeperHiveClientHelper.java
index eeb3cf9..0067ea9 100644
--- a/jdbc/src/java/org/apache/hive/jdbc/ZooKeeperHiveClientHelper.java
+++ b/jdbc/src/java/org/apache/hive/jdbc/ZooKeeperHiveClientHelper.java
@@ -130,7 +130,7 @@ private static void applyConfs(String serverConfStr, JdbcConnectionParams connPa
         // Set http path
         if ((matcher.group(1).equals("hive.server2.thrift.http.path"))
             && !(connParams.getSessionVars().containsKey(JdbcConnectionParams.HTTP_PATH))) {
-          connParams.getSessionVars().put(JdbcConnectionParams.HTTP_PATH, "/" + matcher.group(2));
+          connParams.getSessionVars().put(JdbcConnectionParams.HTTP_PATH, matcher.group(2));
         }
         // Set SSL
         if ((matcher.group(1) != null) && (matcher.group(1).equals("hive.server2.use.SSL"))
-- 
1.7.9.5

